src/metabase/routes.clj

```diff
@@ -31,20 +33,31 @@
 (defn- load-template [path variables]
   (stencil/render-string (load-file-at-path path) variables))
 
+(defn- load-localization []
+  (log/info *locale*)
```

Comment:
We should have some metadata around this log messge. "Starting up with locale:... " or something similar

Reply:
Ah yeah that was added for debugging purposes, but yes, might be good to make it a real log.